app/jobs/decidim/direct_verifications/authorize_users_job.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "decidim/direct_verifications/instrumenter"
+
+module Decidim
+  module DirectVerifications
+    class AuthorizeUsersJob < BaseImportJob
+      class NullSession; end
+
+      def process_users
+        emails.each do |email, data|
+          AuthorizeUser.new(email, data, session, organization, instrumenter).call
```

Comment:
What if we abstract the arguments so that we can pass the processor class to a single job.

Reply:
I realize this would make it harder to call to of these commands in a single loop, like we might want to do with `RegisterUser` and `AuthorizeUser` in the future.